sass/_statistic.scss

```diff
@@ -1,31 +1,53 @@
-
+.nav-side {
+    padding: 1em;
+  }
 .statistic{
     background-color: rgb(255, 254, 254);
-    border: black solid 2px;
     margin: 1rem;
-
+    border: black solid;
+    
     .statistic-navbar {
         text-align: left;
         background-color: $primary-color;
         padding: 2rem;
         width: 20rem;
-        height: 20.6rem;
-        float: auto;
+        height: 20.5rem;
         margin-left: 10rem;
 
-        li a{
-            display: block;
-            color: $white;
-        }
-        li a:hover:not(.active){
-            background-color: $secondary-color;
-            color: $primary-color;
-        }
-    }
-  
-    h2{
+        aside {
+            padding : 1rem;
+            .side-menu {
+              background-color: $primary-color;
+              .title {
+                color: $white;
+                font-weight: bold;
+              }
+              .description {
+                color: white;
+                max-height: 0;
+                overflow: hidden;
+                transition: max-height 0.3s;
+              }
+              /* för description bredvid active */
+              .active + .description {
+                max-height: 200px;
+                transition: max-height 0.2s;
+              }
+              ul {
+                list-style: none;
+                padding: 1em;
+                text-decoration: none;
+                line-height: 1.5em;
+                a {
+                  color: white;
+                }
+              }
+            }
+              }
+            }
+            h2{
         text-align: left;
-        padding-left: 5rem;
+        padding: 1rem 10rem;
     }
     .float-img{
         float: right;
```

Comment:
try: img {
  width: 100%;
  height: auto;
}

Reply:
You do not need
`img {
width: 100%;
height: auto;
}`

It is already defined globally.

Reply:
okay